src/components/map/types.ts

```diff
@@ -182,3 +185,12 @@ export type ParkingType = {
   parkingVehicleTypes: ParkingVehicleTypes;
   totalCapacity: number;
 };
+
+type GeofencingZoneExplanationType = {
+  title: TranslatedString;
+  description: TranslatedString;
+};
+
+export type GeofencingZoneExplanationsType = {
```

Comment:
To be even more precise, this could be called `GeofencingZoneFeatureExplanationsType`, but it feels long enough as is 😅

Reply:
I agree, long enough as is 😅